tsdb/field_validator.go

```diff
@@ -42,6 +43,12 @@ func ValidateAndCreateFields(mf *MeasurementFields, point models.Point, skipSize
 		fieldKey := iter.FieldKey()
 		// Skip fields name "time", they are illegal.
 		if bytes.Equal(fieldKey, timeBytes) {
+			partialWriteError = &PartialWriteError{
+				Reason: fmt.Sprintf(
+					"invalid field name: input field \"%s\" on measurement \"%s\" is invalid. Field \"%s\" has been stripped from point.",
+					"time", string(point.Name()), "time"),
```

Comment:
Don't use the literal `"time"` here, use the `fieldKey` value (use square bracket notation to only pass once to `Sprintf`).  Or maybe rewrite the message to only print the value once.